index.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Signup | Mint.com</title>
+  <link rel="stylesheet" href="assets/style.css">
+  <script src="https://use.fontawesome.com/f220714871.js"></script>
+</head>
+
+<body>
+  <header>
+    <section class="header-text">
+      Already have an Intuit account?<br>
+      <a href="#" class="color-primary">I forgot my User ID or Password</a>
+    </section>
+    <section class="header-button">
+      <button type="button">
+        <i class="fa fa-lock" aria-hidden="true"></i>&nbsp;
+        Sign In</button>
+    </section>
+  </header>
+
+  <main>
+    <section class="logo text-center">
+      <figure>
+        <img src="assets/img/intuit-logo.png" alt="intuit logo" class="intuit-logo"><br>
+      </figure>
+        <span class="mint-logo">
+          <img src="assets/img/mint-logo.png" alt="mint logo">
+          <span>mint</span>
+        </span>
+        <span class="quickbooks-logo">
+          <img src="assets/img/quickbooks-logo.png" alt="quickbooks logo">
+          <span>quickbooks</span>
+        </span>
+        <span class="turbotax-logo">
+          <img src="assets/img/turbotax-logo.png" alt="turbotax logo">
+          <span>turbotax</span>
+        </span>
+    </section>
+
+    <section class="form-heading">
+      <h3 class="text-center">Create an Intuit account</h3>
+      <p class="text-center">One account for everything Intuit, including Mint.<br><a href="/questions" class="color-primary">Learn more</a></p>
+      <form>
+        <p class="form-field">
+          <label for="email">Email address</label><br>
+          <input id="email" type="email" name="email" required/>
+        </p>
+        <p class="form-field">
+          <label for="phone">Phone (recommended)</label><br>
+          <input id="phone" type="tel" name="phone"/>
+        </p>
+        <p class="form-field">
+          <label for="password">Password</label><br>
+          <input id="password" type="password" name="password" required/>
+        </p>
+        <p class="form-field">
+          <label for="confirm-password">Confirm Password</label><br>
+          <input id="confirm-password" type="password" name="confirm-password" required/>
+        </p>
+        <button class="create-account-button background-primary" type="button"><i class="fa fa-lock"
+            aria-hidden="true"></i>&nbsp;Create Account</button>
```

Comment:
- [ ] **[OPTIONAL]** When you use a button in a form, it should have the type attribute submit to work in a  functional form that submits data to the server, it will look like this: `<button class="create-account-button background-primary" type="submit">`